.gitignore

```diff
@@ -56,3 +56,4 @@ Legacy/BinDrivers/X64
 Legacy/BinDrivers/IA32
 Legacy/BootLoader/bin
 Legacy/BootSector/bin
+Utilities/ocvalidate/SampleCustom.plist
```

Comment:
This is added to prevent accidental commits of test materials. It should be removed before merging.

Reply:
I do not mind this. Could be Utilities/ocvalidate/*.plist though.

Reply:
Done.